app/lib/formatter.rb

```diff
@@ -95,6 +95,6 @@ def hashtag_html(match)
   end
 
   def mention_html(match, account)
-    "#{match.split('@').first}<span class=\"h-card\"><a href=\"#{TagManager.instance.url_for(account)}\" class=\"u-url mention\">@#{account.username}</a></span>"
+    "#{match.split('@').first}<span class=\"h-card\"><a href=\"#{TagManager.instance.url_for(account)}\" class=\"u-url mention\">@<span>#{account.username}</span></a></span>"
```

Comment:
Other than default span formatting, does adding <span> here do anything?  Maybe worth adding id=report-username or something like that